src/api/kube.ts

```diff
@@ -1859,12 +1859,26 @@ export class KubeHelper {
     }
   }
 
-  async clusterIssuerExists(name: string): Promise<boolean> {
+  /**
+   * Returns CRD version of Cert Manager
+   */
+  async getCertManagerK8sApiVersion(): Promise<string> {
+    const certManagerCRD = await this.getCrd('certificates.cert-manager.io')
+    if (!certManagerCRD.spec.versions) {
+      // should never happen
+      throw new Error('"spec.versions" is empty')
```

Comment:
return `v1` here as well.
As we can make this function more general.

Reply:
Agree